built-in-ai-playground/src/app/pages/language-detector/language-detector.component.html

```diff
@@ -0,0 +1,141 @@
+<!--
+ Copyright 2025 Google LLC
+ SPDX-License-Identifier: Apache-2.0
+-->
+<div class="container-fluid" xmlns="http://www.w3.org/1999/html">
+
+  <app-page-title [title]="'Language Detector API'" [icon]="'bi-globe2'">
+    <a class="btn btn-light" href="https://github.com/webmachinelearning/translation-api?tab=readme-ov-file#language-detection" target="_blank"><i
+      class="bi bi-file-text"></i> <span class="ms-2">Explainer</span></a>
+    <a class="btn btn-light" href="https://github.com/webmachinelearning/translation-api/pull/38/files#diff-5e793325cd2bfc452e268a4aa2f02b4024dd9584bd1db3c2595f61f1ecf7b985" target="_blank"><i class="bi bi-body-text"></i> <span class="ms-2">Specifications</span></a>
+  </app-page-title>
+
+  <div class="row mb-3">
+    <div class="col-12">
+
+      <app-page-accordion
+        [requirementsStatus]="this.apiFlag.status"
+        (checkRequirementsEvent)="checkRequirements()"
+        [requirements]="getRequirements()"
+      >
+        <div debugInformation>
+          <p>To manually install and uninstall language packs: <span class="code">chrome://on-device-translation-internals/</span>
```

Comment:
Note: Would be great if this was available within Chrome DevTools - developers will need to prepare their apps for the scenario where the device is not capable to run models, where the model is not yet available, and when it's available and will need controls to emulate those (or just delete the models).